it/controllers/IndexControllerISpec.scala

```diff
@@ -1089,7 +1089,7 @@ class IndexControllerISpec extends IntegrationSpecCommonBase {
       summaryCardBody.select("dt").get(0).text shouldBe "Penalty type"
       summaryCardBody.select("dd").get(0).text shouldBe "First penalty for late payment"
       summaryCardBody.select("dt").get(1).text shouldBe "Overdue charge"
-      //      summaryCardBody.select("dd").get(1).text shouldBe "VAT for period 1 January 2021 to 1 February 2021"
+      summaryCardBody.select("dd").get(1).text shouldBe "LPPPenaltyReasonKey"
```

Comment:
This is not correct, most likely because the branch is behind main. Please rebase

Reply:
We will need tests for VAT, ECN, Officers Assessment, Central Assessment

Reply:
Ok so these are the tests to show LPP reason which are ignored now.